### ISSUE TITLE
Error compiling with Ruby 2.0

### ISSUE THREAD

**Issue by joe1chen** [knu/ruby-unf_ext#4]
Using Ruby 1.9.3 is no problem, but compiling with Ruby 2.0 fails.

Here's the output of the /gems/unf_ext-0.0.6/ext/unf_ext/gem_make.out

```
/Users/jc/.rvm/rubies/ruby-2.0.0-p0/bin/ruby extconf.rb 
checking for main() in -lstdc++... yes
checking for ruby/encoding.h... yes
creating Makefile

make
compiling unf.cc
unf.cc:1: error: bad value (native) for -march= switch
unf.cc:1: error: bad value (native) for -mtune= switch
make: *** [unf.o] Error 1
```

**Comment by knu** [comment]
@contentfree The latest version of the gem has binary gems for Windows 64bit/32bit, up to Ruby 2.2.

**Comment by knu** [comment]
Does this still reproduce?  I'm closing this since I haven't been able to reproduce this.  Please refile an issue if you still have the problem, thanks!

**Comment by soundasleep** [comment]
For 0.0.7.4, `sudo apt-get install build-essential g++` solved the issue on Ubuntu 16.04.

**Comment by eugeniomiro** [comment]
on openSUSE (tested on Leap 15.1)
```
$ sudo zypper install -t pattern devel_C_C++
```
